README.md

```diff
@@ -18,8 +18,7 @@ yarn add openai
 
 ## Usage
 
-> [!IMPORTANT]
-> Previous versions of this SDK used a `Configuration` class. See the [v3 to v4 migration guide](https://github.com/openai/openai-node/discussions/217).
+The full API of this library can be found in [API.md file](https://github.com/openai/openai-node/blob/master/api.md). The code below shows how to get started using the chat completions API.
```

Comment:
```suggestion
The full API of this library can be found in [API.md file](https://github.com/openai/openai-node/blob/master/api.md). The code below shows how to get started using the Chat completions API.
```

Reply:
I think you said you wanted this convention right @schnerd ?

Reply:
Gray area, but based on documentation style guide i'm leaning towards keeping this lowercase.